src/Seabug/CallContract.purs

```diff
@@ -261,7 +262,7 @@ buildContractConfig cfg = do
     , logLevel: logLevel
     , suppressLogs: true
     , extraConfig: {}
-    , walletSpec: Nothing
+    , walletSpec: Just ConnectToNami
```

Comment:
We're defaulting to Nami now? Does Gero still work with this change?

Reply:
From what I could see there wasn't any logic implemented to switch the wallet used by the contracts. There's `connectToWallet`, but I don't think that influences the contracts, and the `Nothing` value was just breaking the contracts. I'll double check.

Reply:
#56